recipe/meta.yaml

```
@@ -49,7 +49,7 @@ requirements:
     - arrow-cpp {{ version }}
     - cython >=3
     - numpy 2.0 # [py<313]
```

https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L101
You can remove all these pins and leave only 
```suggestion
    - numpy {{ numpy }}
```